jvb/rootfs/defaults/jvb.conf

```diff
@@ -32,6 +33,11 @@
 {{ $ENV := .Env -}}
 
 videobridge {
+    {{ if $JVB_CC_TRUST_BWE -}}
+    cc {
+        trust-bwe = {{ $JVB_CC_TRUST_BWE | default "true" | toBool}}
```

Comment:
Put the default in the variable definition above, and assign it unconditionally please.

Reply:
I've updated the PR, can you check it out again please?